docs/source/getting_started/installation.rst

```diff
@@ -56,6 +56,10 @@ You can also build and install vLLM from source:
     $ # export VLLM_INSTALL_PUNICA_KERNELS=1 # optionally build for multi-LoRA capability
     $ pip install -e .  # This may take 5-10 minutes.
 
+.. tip::
+
+    Building from source requires quite a lot compilation. To accelerate the compilation, you can install `ccache <https://github.com/ccache/ccache>`_ via either `conda install ccache` or `apt install ccache` . As long as `which ccache` command can find the `ccache`` binary, it will be used automatically by the build system.
```

Comment:
It should be accelerate the subsequent compilation. We should clarify it doesn't help on the first pass.

Reply:
fixed in [38bc622](https://github.com/vllm-project/vllm/pull/5012/commits/38bc6220515ec811186a641cc2d35044b57695a8)